### PR TITLE
Gateway : Cleanup Dead Code / Lazy loading of containers with 

### LINKED ISSUE #1202: Add bundle analyzer and investigate what causing the bundle size increasing

### LINKED ISSUE #1203: Gateway: Try and React Lazy loading if possible

### PR DESCRIPTION
**Pull request by sk-enya**
:clipboard: Add associated issues, tickets, docs URL here.

closes: https://github.com/bobanetwork/boba/issues/1202
closes: https://github.com/bobanetwork/boba/issues/1203

## Overview
 - cleanup vedoa modals, reducers, actions, selector, abi
 -  cleanup old bridging modals
 - cleanig up vote modal and proposal modal
 - cleanup modules and files along with packages
 - clean up exit modal js files
 - removed main menu components which has layerswitcher and other js files
 - migrated connect to tsx
 - removed unused npm packages
 - removed mui styled
 - removed glabal styles js
 - update connect box stylle
 - removed modal for NFT transfer
 - cleanup ui reducer
 - fix: updated reducer state]
 - added lazy loading of modules on routes
 - fix: added analyzer

### CODE REVIEW

**Comment by sk-enya** [comment]
#  Before Lazy Loading: 
Screenshot 2023-09-14 at 12 31 42 PM

# After Lazy Loading :
Screenshot 2023-09-14 at 12 26 15 PM

Bundle Analyzer Output.
Screenshot 2023-09-14 at 12 40 22 PM

File: packages/boba/gateway/config-overrides.js [hunk 1]
```diff
@@ -40,6 +39,15 @@ module.exports = (config, env) => {
       process: 'process',
     })
   );
+  if (process.env.REACT_APP_ENV === 'dev') {
```
Comment (wsdt) [inline packages/boba/gateway/config-overrides.js:42]: might this be interesting to have in a Github action to limit certain thresholds or just have the report for the release attached or similar?

**Review by wsdt — APPROVED** [review]
love it! Left some comments.

File: packages/boba/gateway/config-overrides.js (hunk 1 above)
Comment (wsdt) [reply packages/boba/gateway/config-overrides.js:42]: Furthermore, it may be undesired to open the report on every development startup.

Comment (sk-enya) [reply packages/boba/gateway/config-overrides.js:42]: @wsdt  thank you, would be happy to know the open thoughts on adding this to github action.
- disabled to generate report on each build time.

**Pull request merged**